src/openrct2/object/Object.cpp

```diff
@@ -26,11 +26,6 @@ Object::Object(const rct_object_entry& entry)
     char name[DAT_NAME_LENGTH + 1] = { 0 };
     std::copy_n(entry.name, DAT_NAME_LENGTH, name);
     _identifier = String::Duplicate(name);
-
-    if (IsOpenRCT2OfficialObject())
-    {
-        SetSourceGames({ OBJECT_SOURCE_OPENRCT2_OFFICIAL });
```

Comment:
I mentioned it in my description: `At the same time, this stops marking some custom objects as official, which was a hacky approach that dates from before the JSON objects.`. Basically, it overrode some custom object's source games. I took it out since it's a remnant of an old approach, and also because this PR stops exporting official objects - it would have resulted in an increase of missing objects when opening parks.

Other places calling SetSourceGames simply fill it with the data that was actually in the object and don't override it.

Reply:
Ok, thanks for the explanation :)